lua/nvim-tree.lua

```diff
@@ -377,6 +377,7 @@ local DEFAULT_OPTS = { -- BEGIN_DEFAULT_OPTS
     add_trailing = false,
     group_empty = false,
     highlight_git = false,
+    floating_node = false,
```

Comment:
This option name is not intuitive; perhaps `renderer.full_name`.

Reply:
looks good